### PR TITLE
Update iOS release notes

### PR DESCRIPTION
**Pull request by tjprescott**

### CODE REVIEW

File: releases/2021-02/ios.md [hunk 1]
```diff
@@ -74,11 +68,53 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+
+##### Breaking Changes
+ - Renamed the type `CommunicationUserCredential` to `CommunicationTokenCredential`, as it represents a token.
+ - Communication identifier `MicrosoftTeamsUserIdentifier` property `identifier` renamed to `userId` since identifier was too generic.
+ - Communication identifier `MicrosoftTeamsUserIdentifier` property `id` renamed to `rawId` to represent full MRI.
+ - Communication identifier `PhoneNumberIdentifier` property `id` renamed to `rawId` to represent full MRI.
+ - Removed `CallingApplicationIdentifier` as it is currently unused by any service.
+ - The protocol `CommunicationTokenCredential` has likewise been renamed to `CommunicationTokenCredentialProviding`.
+ - All types that conform to the `CommunicationIdentifier` protocol now use the suffix `Identifier`. For example, the
+    `PhoneNumber` type used to represent a phone number identifier is now named `PhoneNumberIdentifier`.
```
Comment (vcolin7) [inline releases/2021-02/ios.md:84]:
```suggestion
 - All types that conform to the `CommunicationIdentifier` protocol now use the suffix `Identifier`. For example, the `PhoneNumber` type used to represent a phone number identifier is now named `PhoneNumberIdentifier`.
```

File: releases/2021-02/ios.md [hunk 2]
```diff
@@ -74,11 +68,53 @@ If you have a bug or feature request for one of the libraries, please post an is
...
 ## Release highlights
 
-### _Version_(_Link to Changelog_)
-
-#### _Package name_
-
-- Major changes only!
+### 1.0.0-beta.8 ([Changelog](https://github.com/Azure/azure-sdk-for-ios/blob/master/CHANGELOG.md#100-beta8-2021-02-09))
+
+
```
Comment (vcolin7) [inline releases/2021-02/ios.md:73]:
```suggestion

```

File: releases/2021-02/ios.md [hunk 3]
```diff
@@ -74,11 +68,53 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+ - Removed `CallingApplicationIdentifier` as it is currently unused by any service.
+ - The protocol `CommunicationTokenCredential` has likewise been renamed to `CommunicationTokenCredentialProviding`.
+ - All types that conform to the `CommunicationIdentifier` protocol now use the suffix `Identifier`. For example, the
+    `PhoneNumber` type used to represent a phone number identifier is now named `PhoneNumberIdentifier`.
+ - Updated the `CommunicationTokenCredential` initializer that automatically refreshes the token to accept a single
+    `CommunicationTokenRefreshOptions` object instead of multiple parameters.
+
+##### Key Bug Fixes
+- Removing `CommunicationUserCredentialPolicy`, this policy was a duplicate of cores `BearerTokenCredentialPolicy`.  
+  Communication now has new ability to create `BearerTokenCredentialPolicy` using the new `CommunicationPolicyTokenCredential`.
```
Comment (vcolin7) [inline releases/2021-02/ios.md:90]:
```suggestion
- Removing `CommunicationUserCredentialPolicy`, this policy was a duplicate of cores `BearerTokenCredentialPolicy`. Communication now has new ability to create `BearerTokenCredentialPolicy` using the new `CommunicationPolicyTokenCredential`. 
```

Comment (vcolin7) [reply releases/2021-02/ios.md:90]: Is this really a bugfix though? They sound more like a breaking change to me.

File: releases/2021-02/ios.md [hunk 4]
```diff
@@ -74,11 +68,53 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+  Communication now has new ability to create `BearerTokenCredentialPolicy` using the new `CommunicationPolicyTokenCredential`. 
+
+#### Azure Communication Calling
+
+##### New Features
+ - Added ability to join a Teams meeting.
+ - New event on `Call` `OnIsRecordingActiveChanged` to indicate when the recording has been started and stopped and new property `IsRecordingActive` to indicate if currently the recording is active or not.
+
+##### Key Bug Fixes
+ - Fix wrong `callId` on the incoming `Call` object https://github.com/Azure/Communication/issues/164
```
Comment (vcolin7) [inline releases/2021-02/ios.md:99]: Just mentioning the fix is good enough for these notes, we don't need to add a link to the GitHub issue, actually, I would encourage anyone not to add this type of links :P
```suggestion
 - Fix wrong `callId` on the incoming `Call` object.
```

File: releases/2021-02/ios.md [hunk 5]
```diff
@@ -74,11 +68,53 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+ - When placing outgoing call or joining a group call event will be raised `OnCallsUpdated` when call list is updated.
+ - Throw IllegalArgumentException if null camera is passed to constructor of `LocalVideoStream`.
+ - Video freezing in landscape mode https://github.com/Azure/Communication/issues/128
+ - `RendererView` layout is off after a device rotation https://github.com/Azure/Communication/issues/127
+ - `RendererView` is blank when not added to the window right away https://github.com/Azure/Communication/issues/132
+ - `RendererView` Issues when joining a call with a reused `groupId` https://github.com/Azure/Communication/issues/111
```
Comment (vcolin7) [inline releases/2021-02/ios.md:105]: Removing links and rephrasing a bit.
```suggestion
 - Fixed video freezing in landscape mode.
 - `RendererView` layout is no longer off after a device rotation.
 - `RendererView` is no longer blank when not added to the window right away.
 - Fixed `RendererView` issues when joining a call with a reused `groupId`.
```

File: releases/2021-02/ios.md [hunk 6]
```diff
@@ -74,11 +68,53 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+#### Azure Communication Calling
+
+##### New Features
+ - Added ability to join a Teams meeting.
+ - New event on `Call` `OnIsRecordingActiveChanged` to indicate when the recording has been started and stopped and new property `IsRecordingActive` to indicate if currently the recording is active or not.
+
+##### Key Bug Fixes
+ - Fix wrong `callId` on the incoming `Call` object https://github.com/Azure/Communication/issues/164
+ - When placing outgoing call or joining a group call event will be raised `OnCallsUpdated` when call list is updated.
+ - Throw IllegalArgumentException if null camera is passed to constructor of `LocalVideoStream`.
```
Comment (vcolin7) [inline releases/2021-02/ios.md:101]:
```suggestion
 - Throw `IllegalArgumentException` if `null` camera is passed to constructor of `LocalVideoStream`.
```

File: releases/2021-02/ios.md [hunk 7]
```diff
@@ -74,11 +68,53 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+ - `RendererView` layout is off after a device rotation https://github.com/Azure/Communication/issues/127
+ - `RendererView` is blank when not added to the window right away https://github.com/Azure/Communication/issues/132
+ - `RendererView` Issues when joining a call with a reused `groupId` https://github.com/Azure/Communication/issues/111
+
+#### Azure Communication Chat Library
+
+##### New Features
+ - Introduced ChatClient and ChatThreadClient to split operations on threads and operations within a particular thread
+ - Create thread sets repeatability-Request-ID for idempotency if not provided
+ - Introduced MessageContent model to replace string content property
```
Comment (vcolin7) [inline releases/2021-02/ios.md:112]:
```suggestion
 - Introduced `ChatClient` and `ChatThreadClient` to split operations on threads and operations within a particular thread.
 - Create thread sets repeatability-Request-ID for idempotency if not provided.
 - Introduced `MessageContent` model to replace string content property.
```

File: releases/2021-02/ios.md [hunk 8]
```diff
@@ -74,11 +68,53 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+
+##### Breaking Changes
+ - ChatThreadMember renamed to Participant, uses CommunicationUserIdentifier
+ - ChatMessage renamed to Message, uses CommunicationUserIdentifier
+ - ChatThread renamed to Thread, uses CommunicationUserIdentifier
```
Comment (vcolin7) [inline releases/2021-02/ios.md:117]:
```suggestion
 - `ChatThreadMember` renamed to `Participant`, uses `CommunicationUserIdentifier`.
 - `ChatMessage` renamed to `Message`, uses `CommunicationUserIdentifier`.
 - `ChatThread` renamed to `Thread`, uses `CommunicationUserIdentifier`.
```

**Review by vcolin7 — COMMENTED** [review]
In general these look good to me. I added some comments and suggestions for areas we can polish a bit :)

File: releases/2021-02/ios.md (hunk 3 above)
Comment (tjprescott) [reply releases/2021-02/ios.md]: I agree. I moved it to that section and reworded it.

**Review by vcolin7 — APPROVED** [review]

**Pull request merged**